tests/custom_if_prefix/libnetwork_env_var_test.go

```diff
@@ -0,0 +1,66 @@
+package custom_if_prefix
```

Comment:
Add doc.go to the tests directory seems worthwhile to describe the sub-dirs contain tests grouped by environments.